src/Core/Layer/Layer.js

```diff
@@ -53,11 +53,18 @@ export const defineLayerProperty = function defineLayerProperty(layer, propertyN
     }
 };
 
-function GeometryLayer(i) {
+function GeometryLayer(id, object) {
+    if (!id) {
+        throw new Error('Missing id parameter (GeometryLayer must have a unique id defined)');
+    }
+    if (!object || !object.isObject3D) {
+        throw new Error('Missing/Invalid object parameter (must be a three.js Object3D instance)');
+    }
     this._attachedLayers = [];
+    this.object = object;
```

Comment:
Yeah, I tried a few alternatives (`root`, `anchor`), but none of them were really good so I went for the dumbest :)

The alternative is to transform the `GeometryLayer` in a `THREE.Object3D`, that could be added anywhere (another object, a scene, etc).